src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw

```diff
@@ -758,6 +758,18 @@
     <value>Cursor shape</value>
     <comment>Header for a control to select the shape of the text cursor.</comment>
   </data>
+  <data name="Profile_AdjustIndistinguishableColorsNever.Content" xml:space="preserve">
+    <value>Never</value>
+    <comment>An option to choose from for the "adjust indistinguishable colors" setting. When selected, we will never adjust the text colors.</comment>
+  </data>
+  <data name="Profile_AdjustIndistinguishableColorsIndexed.Content" xml:space="preserve">
+    <value>Indexed</value>
+    <comment>An option to choose from for the "adjust indistinguishable colors" setting. When selected, we will adjust the text colors for visibility only when the colors are part of this profile's color scheme's color table.</comment>
+  </data>
+  <data name="Profile_AdjustIndistinguishableColorsAlways.Content" xml:space="preserve">
+    <value>Always</value>
+    <comment>An option to choose from for the "adjust indistinguishable colors" setting. When selected, we will adjust the text colors for visibility (more performance intensive).</comment>

```

Comment:
```suggestion
    <value>Always (More performance intensive)</value>
    <comment>An option to choose from for the "adjust indistinguishable colors" setting. When selected, we will adjust the text colors for visibility.</comment>
```
1. I don't think the localization team is helped by being told that this option is more performance intensive haha
2. _The user_ is definitely helped though. Maybe we should include that in the localized/displayed string